oneshot.d/browser-cleanup-startup-cache

```diff
@@ -1,2 +1,3 @@
 #!/bin/sh
 rm -rf ~/.mozilla/mozembed/startupCache
+rm -rf /home/$USER/.mozilla/captiveportal/startupCache
```

Comment:
I think `/home/$USER` should be replaced with `~`.

Reply:
Yes